datasets/imagenet2012/README.md

```diff
@@ -0,0 +1,1235 @@
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://image-net.org/index.php
+- **Repository:**
+- **Paper:** https://arxiv.org/abs/1409.0575
+- **Leaderboard:** https://paperswithcode.com/sota/image-classification-on-imagenet
+- **Point of Contact:** mailto: imagenet.help.desk@gmail.com  
+
+### Dataset Summary
+
+ILSVRC 2012, commonly known as 'ImageNet' is an image dataset organized according to the WordNet hierarchy. Each meaningful concept in WordNet, possibly described by multiple words or word phrases, is called a "synonym set" or "synset". There are more than 100,000 synsets in WordNet, majority of them are nouns (80,000+). ImageNet aims to provide on average 1000 images to illustrate each synset. Images of each concept are quality-controlled and human-annotated. In its completion, ImageNet hopes to offer tens of millions of cleanly sorted images for most of the concepts in the WordNet hierarchy. 
+
+
+🛑 This dataset requires **manual download** of the ImageNet data tar file (which is about **155GB** in size) [imagenet_object_localization_patched2019.tar.gz](https://www.kaggle.com/competitions/imagenet-object-localization-challenge/data?select=imagenet_object_localization_patched2019.tar.gz) and point datasets library to it by:
+
+```python
+from datasets import load_dataset
+dataset = load_dataset("imagenet2012", data_dir="/path/to/imagenet_object_localization_patched2019.tar.gz")
+```
+
+💡 This dataset provides access to ImageNet (ILSVRC) 2012 which is the most commonly used **subset** of ImageNet. This dataset spans 1000 object classes and contains 1,281,167 training images, 50,000 validation images and 100,000 test images. The version also has the [patch](https://drive.google.com/file/d/16RYnHpVOW0XKCsn3G3S9GTHUyoV2-4WX/view) already applied. For full ImageNet dataset presented in [[2]](https://ieeexplore.ieee.org/abstract/document/5206848), please check the download section of the [main website](https://image-net.org/download-images.php).
+
+### Supported Tasks and Leaderboards
+
+- `image-classification`: The goal of this task is to classify a given image into one of 1000 ImageNet classes. The leaderboard is available [here](https://paperswithcode.com/sota/image-classification-on-imagenet).
```

Comment:
Maybe use the link that filters results to `Imagenet 1k only`? https://paperswithcode.com/sota/image-classification-on-imagenet?tag_filter=171

The link currently provided is all results, 1k and 21k.